.github/workflows/test.yaml

```diff
@@ -122,11 +122,19 @@ jobs:
           # Just Python 3.9 with default settings.
           - python-version: '3.9'
 
+          # Just Python 3.11 with default settings.
+          - python-version: '3.11'
+
           # Newest macOS and newest Python supported versions.
```

Comment:
Please update this comment... 3.10 is no longer the newes Python.

The idea with this comment want to highlight that we only run the tests on a single macOS environemnt.

And that environment is newest macos supported by GitHub  + newest Python supported by Twisted.

In the past we used to run tests on macOS-11 ...but the GitHub environmet was broken , so we stopped running those tests.

Maybe, we can also just remote these tests, and only run python 3.11 on macOS.

I don't think that we need 2 macOS jobs.